schedule.yaml

```diff
@@ -12,7 +12,7 @@ tabs:
     - Mark Welden-Smith
   - title: 'Plenary: Best Practices for Collaborative Development'
     description: Effective teamwork and collaborations
-    time: 9:30 - 10:15
+    time: 9:45 - 10:15
```

Comment:
previous event only scheduled for 9-930 (https://github.com/uwhackweek/schedule-2024/issues/1 - I assume @markweldensmith is presenting. So we actually have a 15 min buffer.

Reply:
I was discussing the scheduling with @aaarendt last week -- it sounded like having 45 minutes for the opening session would help things feel less rushed, and my session is designed for a 30 minute slot anyway. Ultimately, can go with whatever timing works out naturally tomorrow.

Reply:
Yes, I updated the welcome session to be 45 min. Charley needs a bit more time for her activity.